templates/html/bs3/pages/gene_product_details.tmpl

```diff
@@ -107,36 +107,48 @@
   </div>
 </div>
 
-<div class="panel panel-default">
-  <div class="panel-heading">
-    <h3 class="panel-title">Gene Product Associations</h3>
+<!-- Display tabbing. -->
+<ul id="display-tabs" class="nav nav-tabs">
+  <li>
+    <a href="#display-associations-tab" data-toggle="tab">Associations</a>
+  </li>
+  <li>
+    <a href="#display-models-tab" data-toggle="tab">Models</a>
+  </li>
+</ul>
+
+<div class="tab-content">
+  <div id="display-associations-tab" class="tab-pane">
+    <div class="panel panel-default amigo-detail-tab-spacer amigo-live-search-results-unbounder">
+      <div class="panel-body">
+        [% INCLUDE "common/live_search.tmpl" %]
+      </div>
+    </div>
   </div>
-  <div class="panel-body">
-
-    <div id="results-area" class="row">
-
-	<div class="col-xs-12 col-sm-12 col-md-3 col-lg-3">
-	  <div>
-	    <h4>Filter results</h4>
-	    
-	    <div id="accordion" class="bbop-widget-set-live-filters">
-	      Loading...
-	    </div>
-
-	  </div>	  
-	</div>
-
-	<div class="col-xs-12 col-sm-12 col-md-9 col-lg-9">
-	  
-	  <div id="pager" class="bbop-widget-set-live-pager">
-	  </div>
-	  <div id="results" class="bbop-widget-set-live-results">
-	    pending...
-	  </div>
-	</div>
-
+  <div id="display-models-tab" class="tab-pane">
+    <div class="panel panel-default amigo-detail-tab-spacer">
+      <div class="panel-body">
+        <div class="form-group hidden" id="gomodel-select-group">
+          <label for="gomodel-select">Available GO-CAM models</label>
+          <select class="form-control" id="gomodel-select"></select>
+        </div>
+        <div id="gp-gocam-viz-container" class="gocam-viz-container">
+          <button class="btn btn-link btn-collapse-toggle collapsed" type="button" data-toggle="collapse" data-target="#gocam-viz-legend-container" aria-expanded="false" aria-controls="gocam-viz-legend-container">
+            Legend <span class="glyphicon glyphicon-chevron-down" aria-hidden="true"></span> 
+          </button>
+          <div class="collapse" id="gocam-viz-legend-container">
+            <img width="566" id="gocam-viz-legend">
+          </div>
+          </div>
+        <div id="gocam-no-data-message" class="hidden">
+          <p>No GO-CAM models associated with Gene Product [% GP_INFO.internal_id %]</p>
+          <p>To learn more about GO-CAM models visit <a href="https://geneontology.cloud/home">https://geneontology.cloud/home</a>.
```

Comment:
Long story, but the correct URL is: http://geneontology.org/go-cam

Reply:
No problem. Will change.